litho-it/src/test/java/com/facebook/litho/ComponentHostTest.java

```diff
@@ -924,7 +924,7 @@ public void testGetLinkedDrawableForAnimation() {
     assertThat(drawables).hasSize(1);
     assertThat(drawables).contains(d3);
 
-    unmount(2, mountItem2);
+    unmount(1, mountItem2);
```

Comment:
this passes in robolectric3 probably due to Drawable.equals not being strict enough, so `d2.equals(d3) == true` in robo3 but not robo4.

Reply:
This is terrible. Thanks for fixing this.